pkg/middleware/logging/logging.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors"
+	"github.com/ory/viper"
```

Comment:
Please ensure the updated go.sum and go.mod is added to the PR.

Reply:
sure